services/backups/service.py

```
@@ -0,0 +1,733 @@
+
+    def _finish_restore(self, transfer_id: str, capture_id: str, capture: Dict,
+                        plan, ok: bool, message: str, summary: Dict) -> None:
+        now = datetime.now().isoformat()
+        try:
+            self.transfer_model.update(transfer_id, {
+                'status': 'completed' if ok else 'failed',
+                'progress': message,
+                'end_time': now,
+            })
+        except Exception:  # noqa: BLE001
+            pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)